Applications/Telemetry/telemetry.ini

```diff
@@ -8,5 +8,5 @@ schemaname=db.sql
 
 [telemetry]
 units=1
-unit0call=REPLACEME
-unit0id=REPLACEME
\ No newline at end of file
+unit0call= REPLACEME
```

Comment:
Style: do you have a style reference for your `.ini` files? Do you want a space between `=` and `REPLACEME`